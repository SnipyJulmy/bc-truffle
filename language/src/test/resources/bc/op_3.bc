12332131 / 328
9872137218732 / 213
9872137218732 / 213.213
2 * 3 * 4 * 4 * 4 * 4 * 4 * 4 * 4 * 4
1 + 2 * 3 + 1 * 287 * 12 + 131 - 343 * 123 - (1 - 321)
halt
