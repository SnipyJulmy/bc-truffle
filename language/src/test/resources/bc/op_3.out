37597
46348062059
46301760299
393216
-38287
